src/logmessage.cpp

```diff
@@ -27,35 +27,35 @@ namespace g3 {
    // helper for setting the normal log details in an entry
    std::string LogDetailsToString(const LogMessage& msg) {
       std::string out;
-      out.append("\n" + msg.timestamp() + "\t"
```

Comment:
To better demonstrate my motivation, mix std::cout with ColorCoutSink. The new line in the beginning message caused so much trouble. Eg,
``` C++
std::cout<<" Hello"<<endl;
LOG(WARN)<<"Test";
std::cout<<"world"<<endl;
```
The original would look like this

```bash
1.....Hello
2.
3.  LOG: Test world
4. 
```

The new would like 
```bash
1. Hello
2. LOG: Test
3. world
```